apps/qubit/modules/informationobject/actions/editAction.class.php

```diff
@@ -783,4 +787,38 @@ protected function updateChildLevels()
       }
     }
   }
+
+  /**
+   * If identifier mask is enabled, set our new info obj to an identifier generated
+   * from the mask.
+   */
+  private function handleIdentifierFromMask()
+  {
+    if (null === $maskEnabled = QubitSetting::getByName('identifier_mask_enabled'))
```

Comment:
We normally use sfConfig to obtain the settings value, that way you can set a default one ... `sfConfig::get('app_identifier_mask_enabled', false)`. Like this you could remove the exception and make only one if check.